CONTRIBUTING.md

```diff
@@ -69,6 +69,26 @@ Each project card will be in one of the following states:
 
 Note: One or more pull requests must be submitted addressing all user story goals before the issue will be moved to the "under review" status. If any goals or acceptance criteria have not been met, then the user story will be commented on to provide feedback, and the issue will be returned to the "In progress" state.
 
+## Code Review
+
+Contributors are encouraged to [review PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+
+The following are some guidelines to consider when performing code reviews.
+
+- Tone
+   - Do ask open-ended questions and avoid making opinionated statements.
+   - Do keep feedback positive. Consider the effort the author spent writing the PR. Celebrate good work.
+   - Do differentiate between nits and suggestions vs blocking feedback. Limit nits.
```

Comment:
More of a general question in spirit of the new guidance here I really like. I can't find any reference through Google to the old New Relic or other engineering blog post that explains this concept. I did however find this. Is something we can consider to take it further or is this too elaborate?

https://conventionalcomments.org/

```suggestion
   - Do differentiate between nits and suggestions vs blocking feedback. Limit nits. Use [Conventional Comments](https://conventionalcomments.org) to the greatest extent possible.
```